src/BWFMetaEditBundle/Resources/views/Download/macTable.html.twig

```diff
@@ -1,4 +1,12 @@
 <tr>
+    <th rowspan="2" colspan="2"><a href="http://itunes.apple.com/app/bwfmetaedit/id860703858">Mac App Store</a></th>
+    <td class="table-OS" colspan="3">macOS 10.7 to 10.12, x86_64</td>
```

Comment:
[Apple says 10.6](https://itunes.apple.com/us/app/bwf-metaedit/id860703858) actually.
and 10.13 is out, and there should be no change (Apple does not say that there is a problem).
so I think we can say 10.6 to 10.13.

Reply:
"Apple says 10.6 actually" -> Except for dvanalyzer.

Done.

Reply:
ho... Why?
out of topic, if there is no technical reason for that we could change the source code and it would be compatible with 10.6 in next release.

Reply:
Qt links to libc++. Actually I thought I asked 10.7 for all.

I'll test.

Reply:
link to libc++ is for the version we use in order to have QtWebEngine.
Here, we don't use it, and interface is similar to the others, so I guess we could easily support 10.6 by using same config as BWF.

Reply:
We use Qt5.6 for the MAS, linked to libc++.

Reply:
Argh. Curious that BWF build is flagged 10.6.
OK, let's stay with 10.7- everywhere, and we'll see for 10.6 compatibility on MAS if there is a request.

Reply:
For reference: old Qt versions compatible with macOS 10.6 are rejected by Apple, so we can not nowadays provide MacAppStore version for macOS 10.6. the DMG is still built with an old Qt compatible with macOS 10.6.